engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/SchemaRegistryClient.scala

```diff
@@ -0,0 +1,16 @@
+package pl.touk.nussknacker.engine.avro
+
+import org.apache.avro.Schema
+import org.apache.flink.api.common.typeinfo.TypeInformation
+import org.apache.flink.streaming.api.functions.TimestampAssigner
+import pl.touk.nussknacker.engine.kafka.RecordFormatter
+import pl.touk.nussknacker.engine.kafka.serialization.{DeserializationSchemaFactory, SerializationSchemaFactory}
+
+trait SchemaRegistryClient extends Serializable {
+
+  def schemaById(id: Int): Schema
```

Comment:
why here return type is Schema and in others - String?

Reply:
I'v done some changes - can you look at this? :)

Reply:
Gives us a shoot when it will pass the tests :) Serialization problems on Flink can cause a huge changes in design...

Reply:
Now there is no serialization problems ;) I have to fix tests only.